app/views/wiki/new.html.erb

```diff
@@ -29,6 +29,14 @@ See COPYRIGHT and LICENSE files for more details.
 
 <% html_title t("create_new_page") %>
 
+<%=
+  render Primer::OpenProject::PageHeader.new do |header|
+    header.with_title { t("create_new_page") }
+    header.with_breadcrumbs(
+      [{ href: project_overview_path(@project.id), text: @project.name }, t("create_new_page")]
```

Comment:
Here again, we have the question, whether the module "Wiki" should be present in the toolbar. It feels like it is missing:

<img width="800" alt="Bildschirmfoto 2025-04-16 um 09 09 06" src="https://github.com/user-attachments/assets/3b3ad5ef-1fad-48b4-844a-6e361e860cfe" />